sql/core/src/test/resources/sql-tests/inputs/pgSQL/timestamp.sql

```diff
@@ -187,22 +187,21 @@ SELECT '' AS date_trunc_week, date_trunc( 'week', timestamp '2004-02-29 15:44:17
 --   WHERE d1 BETWEEN timestamp '1902-01-01'
 --    AND timestamp '2038-01-01';
 
--- [SPARK-28420] Date/Time Functions: date_part
--- SELECT '' AS "54", d1 as "timestamp",
---    date_part( 'year', d1) AS year, date_part( 'month', d1) AS month,
---    date_part( 'day', d1) AS day, date_part( 'hour', d1) AS hour,
---    date_part( 'minute', d1) AS minute, date_part( 'second', d1) AS second
---    FROM TIMESTAMP_TBL WHERE d1 BETWEEN '1902-01-01' AND '2038-01-01';
-
--- SELECT '' AS "54", d1 as "timestamp",
---    date_part( 'quarter', d1) AS quarter, date_part( 'msec', d1) AS msec,
---    date_part( 'usec', d1) AS usec
---    FROM TIMESTAMP_TBL WHERE d1 BETWEEN '1902-01-01' AND '2038-01-01';
-
--- SELECT '' AS "54", d1 as "timestamp",
---    date_part( 'isoyear', d1) AS isoyear, date_part( 'week', d1) AS week,
---    date_part( 'dow', d1) AS dow
---    FROM TIMESTAMP_TBL WHERE d1 BETWEEN '1902-01-01' AND '2038-01-01';
+SELECT '' AS `54`, d1 as `timestamp`,
```

Comment:
Oops. What I meant was backquoting for the special value like `54`.
For the other values like `year` and `month`, we don't need back quoting.

So, 
1. For the places where PostgreSQL uses `AS ".."`, we can use backquoting.
2. For the places where PostgreSQL uses `AS xxx`, we don't need backquoting, too.

Reply:
Thanks. I will remove the backquoting.

Reply:
@dongjoon-hyun When I removed them, I got the error:
```
-- !query 13
SELECT '' AS `54`, d1 as `timestamp`,
    date_part( 'year', d1) AS year, date_part( 'month', d1) AS month,
    date_part( 'day', d1) AS day, date_part( 'hour', d1) AS hour,
    date_part( 'minute', d1) AS minute, date_part( 'second', d1) AS second
    FROM TIMESTAMP_TBL WHERE d1 BETWEEN '1902-01-01' AND '2038-01-01'
-- !query 13 schema
struct<>
-- !query 13 output
org.apache.spark.sql.catalyst.parser.ParseException

no viable alternative at input 'year'(line 2, pos 30)

== SQL ==
SELECT '' AS `54`, d1 as `timestamp`,
    date_part( 'year', d1) AS year, date_part( 'month', d1) AS month,
------------------------------^^^
    date_part( 'day', d1) AS day, date_part( 'hour', d1) AS hour,
    date_part( 'minute', d1) AS minute, date_part( 'second', d1) AS second
    FROM TIMESTAMP_TBL WHERE d1 BETWEEN '1902-01-01' AND '2038-01-01'
```

Reply:
I would guess, `year` belongs to a functions as well. Probably, this confuses the parser?

Reply:
@dongjoon-hyun I removed backquotes only around names that are not functions.

Reply:
Oh. That sounds like a regression which we made recently.

Reply:
Spark 2.4.3 shows the following and PostgreSQL also allows it. Let me check the logs.
```
spark-sql> select 1 as year;
1
```
Also, cc @maropu .

Reply:
? @MaxGekk . The master branch seems to be same. Could you push your change?
```
spark-sql> select 1 as year;
1
```

Reply:
@dongjoon-hyun Pushed

Reply:
I reverted the backquotes back and opened JIRA for the issue: https://issues.apache.org/jira/browse/SPARK-28767

Reply:
This is because `spark.sql.parser.ansi.enabled` is set to `false` by default:
```
spark-sql> set spark.sql.parser.ansi.enabled=true;
spark.sql.parser.ansi.enabled	true
spark-sql> select 1 as year;
Error in query: 
no viable alternative at input 'year'(line 1, pos 12)

== SQL ==
select 1 as year
------------^^^
```

Reply:
Have you checked that? https://github.com/apache/spark/pull/25114#issuecomment-510537932
I think that's an expected behaviour.